### PR TITLE
chore: vehicle spawn

### PR DESCRIPTION
**Pull request by mockdot**
## Description

- Use modules SpawnVehicle

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

**Review by Manason — APPROVED** [review]

**Review by Manason — CHANGES_REQUESTED** [review]
Fix linting errors

File: server/main.lua [hunk 1]
```diff
@@ -95,14 +95,8 @@ lib.callback.register('police:GetImpoundedVehicles', function()
...
-lib.callback.register('qbx-policejob:server:spawnVehicle', function(source, model, coords, plate)
-    local netId = QBCore.Functions.CreateVehicle(source, model, coords, true)
-    if not netId or netId == 0 then return end
-    local veh = NetworkGetEntityFromNetworkId(netId)
-    if not veh or veh == 0 then return end
-
-    SetVehicleNumberPlateText(veh, plate)
-    TriggerClientEvent('vehiclekeys:client:SetOwner', source, plate)
+lib.callback.register('qbx-policejob:server:spawnVehicle', function(source, model, coords, warp)
+    local netId = SpawnVehicle(source, model, coords, warp)
```
Comment (BerkieBb) [inline server/main.lua:99]: You can return this straight instead of making it a variable

File: client/job.lua [hunk 2]
```diff
@@ -214,17 +219,20 @@ local function spawnHelicopter()
...
-            TriggerEvent("vehiclekeys:client:SetOwner", QBCore.Functions.GetPlate(veh))
-            SetVehicleEngineOn(veh, true, true, false)
-        end, Config.PoliceHelicopter, coords, true)
+        local netId = lib.callback.await('qbx-policejob:server:spawnVehicle', false, Config.PoliceHelicopter, coords)
+        local timeout = 100
+        while not NetworkDoesEntityExistWithNetworkId(netId) and timeout > 0 do
+            Wait(10)
+            timeout -= 1
+        end
+        SetVehicleLivery(veh, 0)
```
Comment (BerkieBb) [inline client/job.lua:228]: You forgot to define the `veh` variable here

File: server/main.lua (hunk 1 above)
Comment (BerkieBb) [reply server/main.lua:99]: Or better yet
```lua
lib.callback.register('qbx-policejob:server:spawnVehicle', SpawnVehicle)
```

**Pull request closed without merging**